packages/@n8n/backend-common/src/logging/logger.ts

```diff
@@ -107,13 +135,24 @@ export class Logger implements LoggerType {
 		}
 	}
 
+	private jsonConsoleFormat() {
+		return winston.format.combine(
+			winston.format.timestamp(),
+			winston.format.metadata(),
+			winston.format.json(),
+			this.scopeFilter(),
+		);
```

Comment:
Should `json` format honor the colorization setting?

Reply:
I'd say no. No tool is rendering this correctly and having ANSI escape sequences within text makes it impossible to read and much harder to parse for setting up alerts.

In my opinion production logs should never be colored because they are not printed to a terminal.